sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/ShareStatistics.java

```diff
@@ -16,6 +17,18 @@
      */
     public ShareStatistics(int shareUsageInGB) {
         this.shareUsageInGB = shareUsageInGB;
+        this.shareUsageInBytes = 0;
```

Comment:
It might be better to default this to -1 so that customers can tell the difference between not set and empty share

Reply:
Done